
try {
    1 / 0
} catch Exception e {
    print 'I got a(n) %s' % match type(e) {
        NameError: 'NameError: %s' % e,
        ZeroDivisionError: 'ZeroDivisionError: %s' % e,
        else: 'Exception: %s' % e,
    }
}

